Core/src/OffConverter/OffModel.cpp

```diff
@@ -24,13 +24,19 @@ void OpenInfraPlatform::Core::OffConverter::OffModel::reset()
 	geometry_.reset();
 }
 
-void OpenInfraPlatform::Core::OffConverter::OffModel::addVertices(std::vector<buw::Vector3d> vertices)
+void OpenInfraPlatform::Core::OffConverter::OffModel::addVertices(const std::vector<buw::Vector3d>& vertices)
 {
 	for (const auto& vertex : vertices)
 		geometry_.vertices.push_back(vertex);
 	geometry_.UpdateBBox();
 }
 
+void OpenInfraPlatform::Core::OffConverter::OffModel::addIndices(const std::vector<uint32_t>& indices)
+{
+	for (const auto& index : indices)
+		geometry_.indices.push_back(index);
```

Comment:
Similar comment applies as with the `addVertices` function.

Reply:
Resolved: 6fad8a0